src/views/Point.js

```diff
@@ -45,35 +45,36 @@ export default function Point() {
 
   const showActions = Just.hasInstance(wallet);
 
-  const goAdmin = useCallback(() => history.push(ROUTE_NAMES.ADMIN), [history]);
+  const goAdmin = useCallback(() => push(names.ADMIN), [history, names]);
```

Comment:
the dependencies for these should be `[push, names]` — as far as I can tell `history` would be undefined here?

Reply:
Oh whoops, bit too hasty here.

_curses at less-loud compile errors_